tests/acceptance/test_member_list.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase
+
+
+class ListOrganizationMembersTest(AcceptanceTestCase):
+    def setUp(self):
+        super(ListOrganizationMembersTest, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=None,
+        )
+        self.team = self.create_team(
+            organization=self.org,
+            name='Mariachi Band'
+        )
+        self.create_member(
+            user=self.user,
+            organization=self.org,
+            role='owner',
+            teams=[self.team],
+        )
```

Comment:
Would you mind adding more members, with different roles, possibly with different invitation status? Just one or two.